### PR TITLE
add some property to jqgrid

### PR DESCRIPTION
**Pull request by LiuXiao617111**
add some property to jqgrid

### CODE REVIEW

**Comment by weswigham** [comment]
Calling @lokeshpeta for review, since this mostly adjusts `jqgrid`.

File: types/fuelux/README.md
Comment (andy-ms) [inline types/fuelux/README.md]: We won't publish a custom README.

File: .github/CODEOWNERS
Comment (andy-ms) [inline .github/CODEOWNERS]: Don't edit this manually, we have a script to update it.

File: types/fuelux/index.d.ts
Comment (andy-ms) [inline types/fuelux/index.d.ts]: Should read `// Type definitions for fuelux 3.16`? (3.16 is the newest version of fuelux according to `npm info fuelux`)

File: types/fuelux/tsconfig.json
Comment (andy-ms) [inline types/fuelux/tsconfig.json]: These should be `true` for all new packages.

File: types/jqgrid/jqgrid-tests.ts
Comment (andy-ms) [inline types/jqgrid/jqgrid-tests.ts]: A test file shouldn't even have a header, please remove it.

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

File: types/fuelux/index.d.ts
Comment (DanielRosenwasser) [inline types/fuelux/index.d.ts]: These just simplify to `any`. Was there something else you meant to type it as?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by RyanCavanaugh** [comment]
@LiuXiao617111 Please address comments from the code reviewers.

File: .github/CODEOWNERS
Comment (LiuXiao617111) [reply .github/CODEOWNERS]: ok

File: types/fuelux/README.md
Comment (LiuXiao617111) [reply types/fuelux/README.md]: ok

File: types/jqgrid/jqgrid-tests.ts
Comment (LiuXiao617111) [reply types/jqgrid/jqgrid-tests.ts]: OK

**Comment by minestarks** [comment]
@LiuXiao617111 Please address the comments from reviewers and fix the test failures. The PR will be closed as stale if no changes. Thanks!

**Comment by minestarks** [comment]
I'm closing this due to test failures and lack of response to requested revisions.

**Pull request closed without merging**
